src/strings/en-gb.json

```diff
@@ -432,6 +432,7 @@
     "ShowYear": "Show the year",
     "ShowIndicatorsFor": "Show indicators for",
     "ShowAdvancedSettings": "Show advanced settings",
+    "ShowDetails": "Show details",
```

Comment:
Languages other than `en-us` should only be updated with Weblate (after PR merge).
```suggestion
```